go.mod

```diff
@@ -47,17 +47,17 @@ require (
 	github.com/operator-framework/operator-marketplace v0.0.0-20190617165322-1cbd32624349
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.11.1
+	github.com/prometheus/client_golang v1.12.1
```

Comment:
Can you explain which dependencies you updated and why?

Reply:
I think it was some command I ran when trying to vendor prometheus/testutils, undid the update